### PR TITLE
Updated color-me module

### PR DESCRIPTION
**Pull request by hfgd123**
Added support for role-icons and did some weird refactoring ESLint forced me to do

### CODE REVIEW

File: modules/color-me/configs/strings.json [hunk 1]
```diff
@@ -26,6 +26,23 @@
...
+      "humanName": {
+        "en": "Role created without icon",
+        "de": "Rolle ohne Icon erstellt"
+      },
+      "default": {
+        "en": "Your role was created successfully, but your role icon was not used, as this requires the guild to be boost level 2 or higher.",
+        "de": "Deine Rolle wurde erfolgreich erstellt, aber dein Rollen-Icon wurde nicht verwendet, da hierfür der Server mindestens Boostlevel 2 besitzen muss."
+      },
+      "description": {
+        "en": "This messages gets send when a booster sucessfully created their custom role, but the guild has not enough boosts to use role icons",
```
Comment (DEVTomatoCake) [inline modules/color-me/configs/strings.json:40]:
```suggestion
        "en": "This messages gets send when a booster successfully created their custom role, but the guild has not enough boosts to use role icons",
```

File: modules/color-me/configs/strings.json [hunk 2]
```diff
@@ -26,6 +26,23 @@
...
+        "de": "Diese Nachricht wird verschickt, wenn ein Booster seine/ihre Custom-Rolle erstellt hat, der Server aber nicht genug boosts hat um Rollenicons zu verwenden"
```
Comment (DEVTomatoCake) [inline modules/color-me/configs/strings.json:41]:
```suggestion
        "de": "Diese Nachricht wird verschickt, wenn ein Booster seine/ihre Custom-Rolle erstellt hat, der Server aber nicht ausreichend Boosts hat, um Rollenicons zu verwenden"
```

File: modules/color-me/configs/strings.json (hunk 1 above)
Comment (hfgd123) [reply modules/color-me/configs/strings.json:40]: I don't like writing good text for users :hAA:

File: modules/color-me/commands/color-me.js [hunk 3]
```diff
@@ -39,19 +49,24 @@ module.exports.subcommands = {
...
                             color: roleColor,
+                            icon: roleIcon,
                             reason: localize('color-me', 'edit-log-reason', {
                                 user: interaction.user.username
                             })
                         }
                     );
-                    await interaction.editReply(await embedType(moduleStrings['updated'], {}, {ephemeral: true}));
+                    if (iconW) {
+                        await interaction.editReply(embedType(moduleStrings['updated'], {}, {ephemeral: true}));
```
Comment (SCDerox) [inline modules/color-me/commands/color-me.js:59]:
```suggestion
                        await interaction.editReply(embedType(moduleStrings['updated']));
``
Not needed here, as the original reply is already ephemeral

**Review by SCDerox — CHANGES_REQUESTED** [review]

File: modules/color-me/commands/color-me.js (hunk 3 above)
Comment (SCDerox) [reply modules/color-me/commands/color-me.js:59]: This is still the case for most calls of editReply

Comment (hfgd123) [reply modules/color-me/commands/color-me.js]: i think i fixed it now

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our Open-Source Developer Program. You can find details in your SCNX organization payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on scnx.app/help.
Cheers ✌️

**Pull request merged**